app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,106 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import flatten from 'lodash/flatten'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('/usr', 'share'),
+  path.join('/usr', 'share', 'ubuntu'),
+  path.join('/usr', 'share', 'gnome'),
+  path.join('/usr', 'local', 'share'),
+  path.join('/var', 'lib', 'snapd', 'desktop')
+]
+
+if (!!process.env.XDG_DATA_DIRS) {
+  appDirs = [
+    ...appDirs,
+    ...process.env.XDG_DATA_DIRS.split(':')
+  ]
+}
+
+// Icon resolutions in priority of checking
+const iconResolutions = [
+  '128x128',
+  '96x96',
+  '64x64',
+  '256x256',
+  '512x512'
```

Comment:
@maximbaz maybe it makes sense. When I set this priority I thought only about productivity in results list. On MacOS I had a problem that rendering and scrolling in long result list was slow with big icons,  and after I set smaller size it became several times faster. But on MacOS these icons are base64-encoded strings, not absolute paths to files, so it also plays a role.

Would be nice to test productivity of big result list with priority from highest to lowest resolution.  I'm checking it in virtual machine so I don't have a lot of applications and vm is slower than real machine.